web/src/actions/apd.js

```diff
@@ -33,7 +34,12 @@ export const WITHDRAW_APD_SUCCESS = Symbol('withdraw apd success');
 export const WITHDRAW_APD_FAILURE = Symbol('withdraw apd failure');
 
 export const SET_SELECT_APD_ON_LOAD = 'SET_SELECT_APD_ON_LOAD';
-export const selectApdOnLoad = () => ({ type: SET_SELECT_APD_ON_LOAD });
+export const selectApdOnLoad = () => (dispatch, getState) => {
```

Comment:
This function is called when a user lands on the `/apd` page before their list of APDs has loaded.  It sets a flag in our application state to load the previously-selected APD once the list of APDs has loaded.  This process could be simplified since we changed how the APD list is loaded, but a story for another time, perhaps.

Anyway, this change just disables setting that flag if the logged-in user is an admin.